addons/stock_account/data/stock_account_demo.xml

```diff
@@ -1,18 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
```

Comment:
remove the file if you remove all the content

Reply:
@abyo-odoo The spec was unclear, we should keep these specific costing method in data, it's useful for testing on runbot